### PR TITLE
UX-1867/Fix provisioner bug in create storage class form

### PR DESCRIPTION
**Pull request by knnguy**
When creating a storage class using in-tree provider for AWS clusters, the YAML is now automatically populated with “provisioner: [kubernetes.io/aws-ebs](http://kubernetes.io/aws-ebs)"

Screen Shot 2022-05-20 at 1 38 29 PM

### CODE REVIEW

File: src/app/plugins/kubernetes/components/storage/storage-classes/add-page/storage-class-config-fields.tsx [hunk 1]
```diff
@@ -28,10 +28,10 @@ const cloudStorageProvisionerValues = switchCase(
   {
     [CloudProviders.Aws]: 'aws-ebs',
     [ClusterCloudPlatforms.EKS]: 'aws-ebs',
-    [CloudProviders.Azure]: 'azure-disk',
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/storage/storage-classes/add-page/storage-class-config-fields.tsx:31]: I dont think we should be changing this. aws-ebs is for aws only. so these values should be correct

Comment (knnguy) [reply src/app/plugins/kubernetes/components/storage/storage-classes/add-page/storage-class-config-fields.tsx:31]: https://platform9.atlassian.net/jira/software/projects/UX/boards/96?selectedIssue=UX-1867

The ticket says AWS, Azure, and GKE

Comment (jruddell) [reply src/app/plugins/kubernetes/components/storage/storage-classes/add-page/storage-class-config-fields.tsx:31]: Ticket is wrong. It's just the `kubernetes/io` prefix. The storage driver is different for these providers

Comment (knnguy) [reply src/app/plugins/kubernetes/components/storage/storage-classes/add-page/storage-class-config-fields.tsx]: Ok, fixed

**Review by jruddell — APPROVED** [review]

**Pull request merged**